src/Components/Button/Documentation/Button.stories.tsx

```diff
@@ -188,7 +188,7 @@ buttonComposedStories.add(
           ]
         }
         popoverType={
-          PopoverType[select('popoverType', mapEnumKeys(PopoverType), 'Solid')]
+          Appearance[select('popoverType', mapEnumKeys(Appearance), 'Solid')]
```

Comment:
I'm finding `popoverType= Appearance[...]` a little confusing :(

Reply:
I could rename the `popoverType` prop to `appearance` for ease of use